var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -84,6 +90,10 @@ class Qt(Package):
     depends_on("libmng")
     depends_on("jpeg")
     depends_on("icu4c")
+    # FIXME:
+    # depends_on("freetype", when='@5.8:') and '-system-freetype'
```

Comment:
can't make Qt pick up `freetype`, don't really want to dig further at this point in time.